stock_inventory_hierarchical/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2013-2016 Numérigraphe SARL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Hierarchical Inventory adjustments",
+    "summary": "Group several Inventory adjustments in a master inventory",
+    "version": "8.0.2.0.0",
+    "depends": ["stock"],
+    "author": u"Numérigraphe,Odoo Community Association (OCA)",
```

Comment:
It is a reasonable precaution because of the accent. Various applications read this file (odoo, the app store, the COA bots...) and I don't expect all to take unicode for granted.